shared-lib/lib/Expression/ExpressionFunctions.js

```diff
@@ -62,6 +62,22 @@ export const ExpressionFunctions = {
 	replaceAll: (str, find, replace) => {
 		return (str + '').replaceAll(find, replace)
 	},
+	decode: (str, enc) => {
+		if (enc === undefined) {
```

Comment:
What about check for an invalid value of `enc`?

Reply:
I have seen at _least_ 13 different acceptable encoding values that is not a definitive list. 
Case is not significant, either. It is coerced into string because a numeric is treated as a buffer offset.
As written, `Buffer.from` throws an error that ends up in the log.
`TypeError [ERR_UNKNOWN_ENCODING]: Unknown encoding: oops, in expression: "decode($(internal:custom_hexval),'oops')"`